src/process_util.rs

```diff
@@ -79,15 +79,15 @@ pub fn capture_fd3_status(mut cmd: Command)
     Ok((status, buf))
 }
 
-pub fn run_and_wait(cmd: &mut Command)
+pub fn run_and_wait(cmd: &mut Command, take_tty: bool)
```

Comment:
Can you explain in docstring here when we need take_tty of true or false? It's not obvious at all. And I think it might hide some bug too.